### PR TITLE
Add checkstyle plugin, fix checkstyle issues

### PR DESCRIPTION
**Pull request by Elishtar**

### CODE REVIEW

File: src/main/java/org/springframework/data/tarantool/core/query/TarantoolRepositoryQueryExecutor.java [hunk 1]
```diff
@@ -7,5 +7,5 @@
  */
 public interface TarantoolRepositoryQueryExecutor {
 
-    Object execute(final Object[] parameters);
+    Object execute(Object[] parameters);
```
Comment (dkasimovskiy) [inline src/main/java/org/springframework/data/tarantool/core/query/TarantoolRepositoryQueryExecutor.java:10]: Looks like we don't protect `parameters` against modification.
@ArtDu could you please look at this changes.

Comment (Elishtar) [reply src/main/java/org/springframework/data/tarantool/core/query/TarantoolRepositoryQueryExecutor.java:10]: @dkasimovskiy If I right understand your point, than it's not issue
https://stackoverflow.com/questions/5380177/final-arguments-in-interface-methods-whats-the-point

**Review by dkasimovskiy — APPROVED** [review]

File: src/main/java/org/springframework/data/tarantool/core/convert/DateConverters.java [hunk 2]
```diff
@@ -26,7 +26,12 @@ public class DateConverters {
         List<Converter<?, ?>> converters = new ArrayList<Converter<?, ?>>();
 
         boolean useISOStringConverterForDate = Boolean
-                .parseBoolean(System.getProperty("org.springframework.data.couchbase.useISOStringConverterForDate", "false"));
+                .parseBoolean(
+                        System.getProperty(
+                                "org.springframework.data.couchbase.useISOStringConverterForDate",
```
Comment (akudiyar) [inline src/main/java/org/springframework/data/tarantool/core/convert/DateConverters.java:31]: @Elishtar Couchbase! 😄 (heh, but now it's easier to remember where I picked this up)

Comment (Elishtar) [reply src/main/java/org/springframework/data/tarantool/core/convert/DateConverters.java:31]: @akudiyar Classic))

**Pull request merged**